importers/vcs/git/git_dao.py

```diff
@@ -400,7 +400,7 @@ def insert_file_renamed(self, repo_id, current_file_id, previous_file_id, file_m
         self._cnx.commit()
         cursor.close()
 
-    def insert_file(self, repo_id, name, ext):
+    def insert_file(self, repo_id, name, ext=None):
```

Comment:
Why the extension is null? Is it possible to retrieve it?
Ideally the process to extract the dependencies should not insert new files, since the files are supposed to be already in the gitana db.